src/main/features/dashboard/views/macro/claimStatus/breachOfPaymentDefendant.njk

```diff
@@ -0,0 +1,77 @@
+      {% endif %}
+    {% endif %}
+    {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+      <p>{{ t('{{ claimantName }} accepted your offer to pay {{ partAdmitAmount }} immediately.',
+          { claimantName: claim.claimData.claimant.name, partAdmitAmount: claim.response.amount }) }}</p>
+      <p>{{ t('Because you didn’t pay within 5 days, they requested the CCJ.') }}</p>
+    {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+      <p>{{ t('They’ve offered to pay by {{ paymentDate }}.', { paymentDate: claim.response.paymentIntention.paymentDate | date }) }}</p>
+    {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
+      <p>{{ t('They’ve offered to pay in instalments.') }}</p>
+    {% endif %}
+
+    {% if claim.claimantResponse.courtDetermination.decisionType !== DecisionType.CLAIMANT %}
+      <p>{{ t('They accepted a new repayment plan determined by the court, based on the financial details you provided.') }}</p>
+    {% else %}
+      <p>{{ t('They suggested a new repayment plan. The court believes you can afford the claimant’s plan, based on the financial details you provided.') }}</p>
+    {% endif %}
+
+    <p>{{ t('{{ claimantName }} accepted your offer to pay {{ partAdmitAmount }} but rejected your repayment plan.',
+        { claimantName: claim.claimData.claimant.name, partAdmitAmount: claim.response.amount }) }}</p>
+    {{ admissionByCCJCommonInfoClaimDetails(claim) }}
+  </div>
+{% endmacro %}
+
+{% macro admissionByCCJAfterPayImmediatelyDeadlineClaimDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('A County Court Judgment (CCJ) has been issued against you') }}</h2>
+    {% if claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
+      <p>{{ t('You agreed to pay the full {{ amount }} immediately.', { amount: claim.response.amount }) }}</p>
+      {% elif claim.response.responseType === domain.ResponseType.PART_ADMISSION %}
+      <p>{{ t('{{ claimantName }} accepted your offer to pay {{ partAdmitAmount }} immediately.',
+          { claimantName: claim.claimData.claimant.name, partAdmitAmount: claim.response.amount }) }}</p>
+    {% endif %}
+    <p>{{ t('Because you didn’t pay within 5 days, they requested a CCJ against you.') }}</p>
+
+    {{ admissionByCCJCommonInfoClaimDetails(claim) }}They requested the judgment because you
+  </div>
+{% endmacro %}
+
+{% macro CCJAfterDefendantDidNotSignSettlementAgreementClaimDetails(claim) %}
+  <div class="status-block">
+    <h2 class="heading-small">{{ t('A County Court Judgment (CCJ) has been issued against you') }}</h2>
+    {% if claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
+      {% elif claim.response.responseType === domain.ResponseType.PART_ADMISSION %}
+    {% endif %}
```

Comment:
Missing some work here

Reply:
@robertparkinson if you still blocked then please let me know?